resources/views/pages/messages/create.blade.php

```diff
@@ -1,18 +1,28 @@
 <?php
 
 use App\Models\Message;
-use function Laravel\Folio\{middleware, name};
+use Illuminate\View\View;
 
-middleware(['auth', 'can:create,' . Message::class]);
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:create,' . Message::class]); // TODO add 'verified' middleware
```

Comment:
I get a `Argument #2 ($arguments) must be of type array, int given` error after submitting a newly created message.

Reply:
Resolved in latest.